app/src/main/java/org/eyeseetea/malariacare/domain/boundary/repositories/IProgramRepository.java

```diff
@@ -8,4 +8,10 @@
     void saveUserProgramId(Program program);
 
     Program getUserProgram();
+
+    boolean checkLastDownloadedProgramMedia(String program);
```

Comment:
I think that program associate to media should be program associated to the user then these methods are not necessary and remove media files is not a ProgramRepository responsibility